dbt_subprojects/dex/macros/models/_project/uniswap_downstream_trades.sql

```diff
@@ -0,0 +1,380 @@
+{% macro uniswap_downstream_trades(
+    blockchain = null
+    )
+%}
+
+{% set v4_models = [
+    ref('uniswap_v4_arbitrum_base_trades')
+    , ref('uniswap_v4_avalanche_c_base_trades')
+    , ref('uniswap_v4_base_base_trades')
+    , ref('uniswap_v4_blast_base_trades')
+    , ref('uniswap_v4_bnb_base_trades')
+    , ref('uniswap_v4_ethereum_base_trades')
+    , ref('uniswap_v4_ink_base_trades')
+    , ref('uniswap_v4_optimism_base_trades')
+    , ref('uniswap_v4_polygon_base_trades')
+    , ref('uniswap_v4_unichain_base_trades')
+    , ref('uniswap_v4_worldchain_base_trades')
+    , ref('uniswap_v4_zora_base_trades')
+] %}
+
+{% set bunni_models = [
+    {"source": source('bunni_v2_base', 'bunnihook_evt_swap'), "chain": "base"}
+    , {"source": source('bunni_v2_arbitrum', 'bunnihook_evt_swap'), "chain": "arbitrum"}
+    , {"source": source('bunni_v2_ethereum', 'bunnihook_evt_swap'), "chain": "ethereum"}
+    , {"source": source('bunni_v2_unichain', 'bunnihook_evt_swap'), "chain": "unichain"}
+    , {"source": source('bunni_v2_bnb', 'bunnihook_evt_swap'), "chain": "bnb"}
+] %}
```

Comment:
i would recommend removing these lists from within the macro. otherwise, each time new chains are added, all chains will run since macro is modified.

we should set it up so that macro rarely changes (bug fixes only) and simply adding new models with `blockchain` param is needed